app.js

```diff
@@ -0,0 +1,48 @@
+let library = [];
+
+function getInput() {
+  const books = {};
+  books.title = document.getElementById('title').value;
+  books.author = document.getElementById('author').value;
+  return books;
+}
+
+function removeBook(title) {
```

Comment:
Remove book functionality removes all the books when there are books with the same title and author name. It should remove the book which user wants to remove rather than remove all the same books. Please fix the issue.